hedera-node/hedera-consensus-service-impl/src/test/java/com/hedera/node/app/service/consensus/impl/test/ReadableTopicStoreTest.java

```diff
@@ -79,7 +79,7 @@ void objectMethodsWorks() {
                 false);
 
         final var expectedString =
```

Comment:
I don't know why we have tests for toString (or why we implement the method either). It shouldn't ever be called in production (performance impact), and in development we can use a debugger. It isn't clear to me why we have this code.